app/controllers/CreateSubscriptionAndUpdateEnrolment.scala

```diff
@@ -68,6 +68,7 @@ trait CreateSubscriptionAndUpdateEnrolment extends Logging {
           } else {
             Future.successful(Redirect(routes.PreRegisteredController.onPageLoad(false)))
           }
+        case _ => Future.successful(Redirect(routes.PreRegisteredController.onPageLoad(false)))
```

Comment:
THis should go to thisisaproblemcontroller

Reply:
changes done as per comment